lib/heroku/application.rb

```diff
@@ -29,16 +29,11 @@ def deploy_branch(tarball)
     end
 
     def fork_into(application)
-      Heroku::CLT.execute(
-        api_key: api_key,
-        app: name,
-        cmd: "da-fork",
-        args: "--copy-pg-data #{application.name}"
```

Comment:
Yep, I definitely included it. However, I have committed some additional stuff because, apparently, PATH is not being loaded and a bundled version of `heroku` is being loaded instead. Please, stay tuned on master for further information :ear: